app/views/admin/index.scala.html

```diff
@@ -823,11 +825,146 @@ <h1>Label Search</h1>
                     </form>
                     <button type="submit" id="submit" class="btn btn-primary">Submit</button>
                 </div>
+                <div id="tabs-7" class="tab-pane">
+                    <h1>Teams</h1>
+                    <table id="teams-table" class="table table-striped">
+                        <thead>
+                            <tr>
+                                <th>Team Name</th>
+                                <th>Description</th>
+                                <th>Status</th>
+                                <th>Visibility</th>
+                            </tr>
+                        </thead>
+                        <tbody id="teams-body">
+                        </tbody>
+                    </table>
                 </div>
             </div>
         </div>
-
     </div>
+    <script>
+        $(document).ready(function() {
+            $('#teams-tab').on('click', function() {
```

Comment:
This should be in public/javascripts/Admin/src/Admin.js where there's similar functionality for the other tabs/tables

Reply:
I haven't looked at the code for the other tables, but we should be mirroring what's done in those where appropriate!